packages/falso/src/lib/credit-card.ts

```diff
@@ -81,5 +86,10 @@ export function randCreditCard<Options extends CreditCardOptions = never>(
     };
   };
 
-  return fake(factory, options);
+  return fake(factory, options, { uniqueComparer: checkUnique });
```

Comment:
Same comment as before, either convert to function or move the const decleration before the `randCreditCard ` def.